### ISSUE TITLE
[Bug]: Quake Gloves shrink comically on death.

### ISSUE THREAD

**Issue by Bumbo-Walrusoni** [openfortress/public-issue-tracker#97]
### Describe the bug.

When you die in a way that can dismember the Mercenary, the Quake Gloves can appear inside out or 'dehydrated' if you have them equipped when doing so.

### Version

rev20

### What server did this occur on?

Local Server (Applies everywhere)

### Reproduction steps

```bash
1. Load on any server with the Quake Gloves equipped in your loadout slot.
2. Die in anyway that causes dismemberment of the Merc. (Initial testing was done with blast damage.)
3. Presto, the mercs hands are inverted wearing the gloves.
...
```

### Which operating systems have you used?

Windows

### Expected behavior

Presumably, the hands would become stubs , or appear fine, like this.

Instead, they appear to be contorted in someway.

### Additional context

With Quake Gloves: ![20240622165417_1](https://github.com/openfortress/public-issue-tracker/assets/86118282/194bdeb9-37b9-4d39-bb4d-c40f352b8360)

With other gloves (no gloves in this example): ![20240622165524_1](https://github.com/openfortress/public-issue-tracker/assets/86118282/d3d9d573-d638-4c38-a5bb-45faa1ddc6de)

**Comment by KaydemonLP** [comment]
Our gore system currently mostly only supports the default mercenary.
This happens with every cosmetic that replaces a body part.
We have plans to iron these out later but not for now.

TLDR: We're aware, and we'll fix it later.
